.github/workflows/ReleaseDgc.yml

```diff
@@ -12,7 +7,8 @@ jobs:
       - uses: actions/checkout@v2
       - name: Run tests
         run: cargo test --verbose
-      - name: Release on Crates.io
+      - name: Release Dgc on Crates.io
         run: |
+          cd dgc
           cargo login ${{ secrets.CARGO_TOKEN }}
           cargo publish
```

Comment:
Will this one publish all the packages independently?

Reply:
It is not possible to do a `cargo publish` directly on a workspace according to the docs.
But it should be nice to trigger on specific tag releases in order to have a `cargo publish` for the released crate. Do you have any hints?

Reply:
Thanks to @guerinoni, I finally found a way to restore the release automation depending on the tag used :tada: (it was easy, but I was searching in the wrong direction).

Reply:
:D happy to help

Reply:
We could change the pipeline to cd into the individual packages and publish them one by one.

I am not sure if that's what we want though.

Would it make sense that once a package is changed we republish all of them (and keep the versions in sync)?

I have a feeling this might make sense if we change the core but not the other way around (we won't release a new version of the core of we just change the Italian lib).

What do you think?

Reply:
The problem with keeping the version synced is that, if you need to bump the Italian crate because of a breaking change, then you will need to do a breaking semver bump for the `dgc` crate as well. But one of the reasons to split into different crates was exactly to avoid this kind of problem, am I wrong?

My 2 cents: let's keep simple things simple. If nothing has changed in `dgc`, there is no reason to bump its version :blush: .

Reply:
Makes tons of sense. I generally prefer to keep libs in sync in a monorepo project just to avoid the mess of "this version of this works with... which other version of the other lib, now?"

But I don't think this will be a big problem here since the dependency tree should be quite flat.

So 100% on keeping this simple and adjust as we learn more.

Let's 🚢  it!